.buildkite/auditbeat/auditbeat-pipeline.yml

```diff
@@ -32,6 +32,10 @@ steps:
         command: |
           cd auditbeat
           mage build unitTest
+        retry:
+          automatic:
+            - exit_status: 1
```

Comment:
Is there a reason for wanting to only retry when the exit code is `1` and nothing else?

I'd think that we should retry in all cases except when `0` is returned, which is covered by the default value of `*` as [described by the docs](https://buildkite.com/docs/pipelines/command-step#retry-attributes-automatic-retry-attributes).